.github/workflows/ci.yml

```diff
@@ -19,15 +19,19 @@ env:
 
 jobs:
 
+  # Check style compliance of JavaScript code.
+
```

Comment:
This comment also seems to explain the obvious to me, but maybe i've been looking at this code too long :older_adult:

Reply:
True, it is just a comment describing the intention of this step
and everyone who is looking at the last step of the action may guess it by first sight.

With these comments you didn't have to ;)
it feels more like a headline of an article.

Side note: at first, I've just added it to be consistent with the rest of the file (new line, comment, newline, action).